crates/cargo-wdk/README.md

```diff
@@ -0,0 +1,101 @@
+# cargo-wdk
+
+A Command-Line Interface (CLI) utility to create and build Windows driver projects written in Rust. 
+`cargo-wdk` is a cargo extension (plugin) with "cargo-like" commands to
+- create new projects from scratch 
+- build new or existing projects
+
+**NOTE**: `cargo-wdk` utility is designed to work with projects that depend on the Windows Driver Kit (WDK) and the "windows-drivers-rs" platform
+
+## Installation
+
+To install cargo-wdk, you need to have Rust and Cargo installed on your system. 
+You can install Rust and Cargo by following the instructions on the [Rust website](https://www.rust-lang.org/tools/install).
```

Comment:
We can assume the user will have Rust already installed. So I think we can remove this.

Reply:
That could be helpful for those new to Rust but a distraction for others. That said, I don't have a very strong opinion. If we want to retain it we should replace "Rust and Cargo" with just "Rust" as cargo is always installed with it.